usaspending_api/disaster/tests/integration/test_disaster_agency_spending.py

```diff
@@ -1,16 +1,22 @@
+import datetime
+
 import pytest
 
 from rest_framework import status
 from usaspending_api.search.tests.data.utilities import setup_elasticsearch_test
-
+from usaspending_api.submissions.models import DABSSubmissionWindowSchedule
 
 url = "/api/v2/disaster/agency/spending/"
 
 
 @pytest.mark.django_db
-def test_basic_success(client, disaster_account_data, elasticsearch_award_index, monkeypatch, helpers):
-    setup_elasticsearch_test(monkeypatch, elasticsearch_award_index)
+def test_basic_success(client, disaster_account_data, elasticsearch_account_index, monkeypatch, helpers):
+
     helpers.patch_datetime_now(monkeypatch, 2022, 12, 31)
+    bad_date_window = DABSSubmissionWindowSchedule.objects.get(id=2022071)
```

Comment:
There has got to be a better way to setup the mock data

Reply:
Is this just pulling in a `DABSSubmissionWindowSchedule` that was created elsewhere in a pytest fixture? If so, seems ok (the ID format uses a deterministic date+boolean structure)

Reply:
The concern wasn't based on confusion but how it is altering a record. It isn't very robust and leaves the test open to testing behavior the code isn't expecting and/or break for potential valid future changes. Also isn't a pattern worth repeating. 

Not a blocker to this PR